### PR TITLE
Week 11

### PR DESCRIPTION
**Pull request by payasbhrigu**

### CODE REVIEW

**Comment by payasbhrigu** [comment]
**Step-Counter using redux**
StepCounter

**Comment by payasbhrigu** [comment]
@anilrao316

**Comment by anilrao316** [comment]
Add recordings

**Comment by anilrao316** [comment]
Also add test cases for any of the assignments

**Comment by anilrao316** [comment]
rest looks good!

File: Week-11/step-counter/src/StepCounter-Context.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { StepContext } from "./StepContext";
+
+export default function StepCounter(){
+
+    const [count,setCount] = useContext(StepContext)
+
+    function handleStepReset() {
+        setCount(0)
+    }
+
```
Comment (anilrao316) [inline Week-11/step-counter/src/StepCounter-Context.js:11]: Looks good!

Comment (anilrao316) [reply Week-11/step-counter/src/StepCounter-Context.js:11]: we can merge the PR

**Pull request merged**